mscore/prefsdialog.cpp

```diff
@@ -1426,7 +1429,9 @@ void PreferenceDialog::apply()
             else if (
                (wasJack != nowJack)
                || (preferences.getBool(PREF_IO_ALSA_USEALSAAUDIO) != alsaDriver->isChecked())
+#ifdef USE_PORTAUDIO
                || (preferences.getBool(PREF_IO_PORTAUDIO_USEPORTAUDIO) != portaudioDriver->isChecked())
+#endif
                || (preferences.getBool(PREF_IO_PULSEAUDIO_USEPULSEAUDIO) != pulseaudioDriver->isChecked())
       #ifdef USE_ALSA
```

Comment:
While at it: please unindent this too

Reply:
It was indented. There are global identation problem around the all source. There are 4, 5 spaced tabs. Is there any plan to apply autoindentation tools to conform some programming style?

Reply:
no

Reply:
I've fixed it myself

Reply:
Let me know whether you're happy with my changes